hypha/apply/funds/management/commands/seed_community_lab_application.py

```diff
@@ -48,9 +48,9 @@ def create_community_lab_form(self):
             {"type": "multi_file", "value": {"field_label": "Upload", "help_text": "", "required": ""}, "id": "b3af7aac-3439-45fa-9573-518f82f5cd6c"},
             {"type": "text_markup", "value": "<h3>I acknowledge</h3>", "id": "b4bbe32a-058d-44b0-aaea-bddc70674277"},
             {"type": "checkbox", "value": {"field_label": "My application will be dismissed if it does not fit within OTF\'s mission, values, principles statements.", "help_text": "", "default_value": ""}, "id": "1248f597-2f18-4b16-8f96-63912e5197c5"},
-            {"type": "text_markup", "value": "Read our <a href=\"\/about/program\">mission, values, and principles</a>.", "id": "67b65f65-5d9f-4152-9c0b-d980d9944e3d"},
+            {"type": "text_markup", "value": "Read our <a href=\"\\/about/program\">mission, values, and principles</a>.", "id": "67b65f65-5d9f-4152-9c0b-d980d9944e3d"},
```

Comment:
This does not seem correct and looks false positive for deprecation warnings. Can you please check and provide more context?

Reply:
`\/` is a deprecation warning as shown below. raw strings are not used since there is `\"` in the string and using raw literal prefix will cause behavior change.

```
python -Wall
Python 3.9.0b5 (default, Jul 20 2020, 20:46:56) 
[GCC 7.5.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> '\/'
<stdin>:1: DeprecationWarning: invalid escape sequence \/
'\\/'

>>> r"\"\/abc" # double quote is already escaped with backslash so using raw literal prefix
'\\"\\/abc'
>>> "\"\/abc"
<stdin>:1: DeprecationWarning: invalid escape sequence \/
'"\\/abc'
```

Reply:
Thanks for the examples.